bin/node/runtime/src/lib.rs

```diff
@@ -441,6 +441,9 @@ parameter_types! {
 	pub const MaxIterations: u32 = 10;
 	// 0.05%. The higher the value, the more strict solution acceptance becomes.
 	pub MinSolutionScoreBump: Perbill = Perbill::from_rational_approximation(5u32, 10_000);
+	// The unsigned solution is allowed to take almost the entire block. Note that it is also
+	// operational.
+	pub MaximumUnsignedWeight: Weight = Perbill::from_percent(90) * MaximumBlockWeight::get();
```

Comment:
You will probably still be limitted by `MaximumExtrinsicWeight` before that. I think we should use a `min` of these two.

Reply:
Also IMHO `MaximumUnsignedWeight` is not self-explainatory enough, without the context of reading PR description I'd have no idea what you mean. Maybe `MaxUnsignedSolutionTransactionWeight`?

Reply:
Then this is a big problem. Ideally, because this is operational, it should be exempt from `MaximumExtrinsicWeight`. Otherwise this won't work at all :( we want this take a very large portion of the block weight, definitely more than `MaximumExtrinsicWeight`.

Reply:
You're right, `Operational` are exempt:
https://github.com/paritytech/substrate/blob/td-mmr/frame/system/src/extensions/check_weight.rs#L72

It should be fine then, we can increase it even further, although we might want to introduce a static assert to make sure it satisfies:
```
MaximumUnsignedWeight < MaxBlockWeight * (1 + (1 - AvailableBlockRatio)) - SomeSaneThreshold
```

Reply:
Indeed it seems like operational ones are not limited by this, which is exactly what I want.

Reply:
Though I am limited by 

```
let operational_limit = operational_limit.saturating_sub(T::BlockExecutionWeight::get());
```

Meaning that This value should not be more than `T::MaximumBlockWeight - T::BlockExecutionWeight`.

Reply:
No, afair the `dispatch_ratio` for operational is `1 + (1 - NormalDispatchRatio)`, so you are limited by the formula I've posted above - `SomeSaneThreshold` has to include `BlockExecutionWeight` and `BaseExtrinsicWeight`.

Reply:
> Also IMHO MaximumUnsignedWeight is not self-explainatory enough, without the context of reading PR description I'd have no idea what you mean. Maybe MaxUnsignedSolutionTransactionWeight?

Was trying to keep the name short 🙈  I would argue that given staking submitting just one type of unsigned solution, it is okay. I will settle for now with `OffchainSolutionWeightLimit`. I think this is short and pretty accurate about what this is.